assets/src/edit-story/elements/text/display.js

```diff
@@ -147,7 +132,7 @@ function TextDisplay({
             <BackgroundSpan
               {...props}
               dangerouslySetInnerHTML={{
-                __html: draftMarkupToContent(content, bold),
+                __html: draftMarkupToContent(content),
```

Comment:
That method looks rather misnamed. It's really just `cleanContent()` or such.

Reply:
Yes, it's about to be completely removed (or reduced). I just hadn't gotten around to it yet.

Reply:
Added to todo as "cleanup".

Reply:
Resolving this since we have another thread open on this exact item.